ethereumj-core/src/main/java/org/ethereum/solidity/compiler/Solc.java

```diff
@@ -25,6 +27,21 @@ private Solc() {
     }
 
     private void init() throws IOException {
+        SystemProperties defaultProps = SystemProperties.getDefault();
```

Comment:
Can you please also make `SolidityCompiler` a Spring @Component to pick up the `SystemProperties` from the AppContext? We eliminated all the static references to it recently.